numpy/ma/core.pyi

```diff
@@ -413,8 +415,8 @@ class MaskedArray(ndarray[_ShapeType_co, _DType_co]):
     def __ifloordiv__(self, other): ...
     def __itruediv__(self, other): ...
     def __ipow__(self, other): ...
-    def __float__(self): ...
-    def __int__(self): ...
+    def __float__(self) -> float: ...
+    def __int__(self) -> int: ...
```

Comment:
These two are also defined in `ndarray` (`_ArrayOrScalarCommon` to be specific), so removing them is also an option

```suggestion
```

Reply:
well I'm always happy to delete code